execute_stacksentinel_run_files.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python3
```

Comment:
This is a new script and following functions from _process_utilities are moved to here  and got modified :
get_run_files
set_memory_defaults
submit_isce_jobs